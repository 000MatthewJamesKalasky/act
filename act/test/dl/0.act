function test (pint x) : pint;

bool x[test(4)];
bool y[test(4)];

x = y;
