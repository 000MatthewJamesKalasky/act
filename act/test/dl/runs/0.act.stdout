got 1 args
  arg[0] = 4
got 1 args
  arg[0] = 4
